public/index.html

```diff
@@ -7,7 +7,7 @@
 </head>
```

Comment:
Did you also want to rename the `title` attribute in `head`? Currently it's "AICS 3D Volume Viewer"

Reply:
May need to do an additional sweep for "Volume Viewer" keywords